.github/workflows/app.yml

```diff
@@ -34,14 +34,18 @@ jobs:
       - uses: Swatinem/rust-cache@v1
 
       - uses: nttld/setup-ndk@v1
+        id: setup-ndk
         with:
           ndk-version: r23c
+          add-to-path: false
```

Comment:
why not adding it to the path?

Reply:
Before I touched this part, `ANDROID_NDK_HOME` env variable was not working.
So I thought it is reasonable to delete this variable, because it is not working.
And [the tutorial that I followed](https://github.com/Haivision/srt/actions/runs/1312237281/workflow) is using `add-to-path: false` too.
If you want, I will try to delete `add-to-path: false`.

Reply:
the example you are linking at has an error (as github says), but if it works, fine with me. was just wondering if that had a special reason... as it means we don't have `adb` and stuff in `$PATH` and that makes other things harder usually..

Reply:
When ndk r23 is updated to fix this error in the future and someone is about to remove this limit, he could notice env variable problem with seeing `add-to-path: false` and `ANDROID_NDK_HOME`.